sdk/appconfiguration/Azure.ApplicationModel.Configuration/src/AuthenticationPolicy.cs

```diff
@@ -60,7 +60,7 @@ private async Task ProcessAsync(HttpPipelineMessage message, bool async)
                 var host = uri.Host;
                 var pathAndQuery = uri.PathAndQuery;
 
-                string method = HttpPipelineMethodConverter.ToString(message.Request.Method);
+                string method = message.Request.Method.Method;
```

Comment:
Should we rename `Method` to `Name` or `Verb`?  I think the common use case will end up being `method.Method`.

Reply:
Yeah, I thought about it, but that's what HttpClientMethod also did and almost no-one would access `Method` property anyway.